ament_cmake_uncrustify/cmake/ament_uncrustify.cmake

```diff
@@ -11,12 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 #
 # Add a test to check the code for compliance with uncrustify.
 #
+# The default configuration file used for uncrustify is located at
+# configuration/ament_code_style.cfg within the uncrustify directory
+# The default configuration file can be overridden by the
+# argument 'CONFIG_FILE'.
+#
 # :param TESTNAME: the name of the test, default: "uncrustify"
 # :type TESTNAME: string
+# :param CONFIG_FILE: the path of the configuration file for
+#                     uncrustify to consider
```

Comment:
Please avoid the alignment and intend this line the same as for the `MAX_LINE_LENGTH` argument.

Reply:
Noted. The indentation has been corrected.